.github/workflows/build_and_test.yml

```diff
@@ -979,6 +979,22 @@ jobs:
       VERSIONS: v14 v15 v16 v17
 
     steps:
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-central-1
+          role-to-assume: ${{ vars.DEV_AWS_OIDC_ROLE_ARN }}
```

Comment:
Hmm, are you sure about `DEV_AWS_OIDC_ROLE_ARN` Shouldn't it be something with "PROD"?

Reply:
Yes: We already have the AWS-prod credentails further down.